crates/adapters/src/format/parquet/mod.rs

```diff
@@ -264,7 +266,12 @@ pub fn relation_to_parquet_schema(
             SqlType::Time => DataType::Time64(TimeUnit::Nanosecond),
             // DeltaLake only supports microsecond-based timestamp encoding, so we just
             // hardwire that for now.  We can make it configurable in the future.
-            SqlType::Timestamp => DataType::Timestamp(TimeUnit::Microsecond, None),
+            // FIXME: Also, the timezone should be `None`, but that gets compiled into `timezone_ntz`
+            // in the JSON schema, which Databricks doesn't fully support yet.
+            SqlType::Timestamp => DataType::Timestamp(
```

Comment:
They must have a timezone?
In SQL TIMESTAMP is a different type from TIMESTAMP WITH TIMEZONE.
BTW: I added the later to Calcite, it will ship in the next release, but I didn't implement all the functions.

Reply:
The no-time-zone version of timestamp is in preview:
https://docs.databricks.com/en/sql/language-manual/data-types/timestamp-ntz-type.html
When I tried feeding it to Databricks, it got rejected.